.build/build.ps1

```diff
@@ -184,7 +230,7 @@ task PublishNugetPackages -If($PublishNugetPackages) {
 }
 
 # Synopsis: Build the project.
-task Build Init, Compile, UnitTests, BuildNugetPackages, PublishNugetPackages
+task Build Init, Compile, UnitTests, BuildNugetPackages, PublishNugetPackages, CreateTransferArtifact
```

Comment:
We seem to be publishing the NuGet package, yet also generating the transfer artefact, presumably because some subsequent builds will run that depend on the transfer artifact. It would probably be better to not publish here, and do the publish in a separate build configuration that depends on **all** of the build configurations.